lib/screens/home.dart

```diff
@@ -35,10 +35,12 @@
 library;
 
 import 'package:flutter/material.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
 
 import 'package:intl/intl.dart';
 import 'package:solidpod/solidpod.dart';
-
+import 'package:solidpod/src/widgets/setting.dart';
```

Comment:
I think this (setting.dart) should become a popup widget. Then all of the code here should be replaced by a button "Change Key" which calls SolidKeyChange() to popup a little window with the interface you have now (old, new, repeat text entry and two buttons CHANGE and CANCEL).

Reply:
> I think this (setting.dart) should become a popup widget. Then all of the code here should be replaced by a button "Change Key" which calls SolidKeyChange() to popup a little window with the interface you have now (old, new, repeat text entry and two buttons CHANGE and CANCEL).

got it, working on it now :)